src/runtime_src/core/common/api/aie/xrt_graph.cpp

```diff
@@ -228,6 +228,76 @@ send_exception_message(const char* msg)
   xrt_core::message::send(xrt_core::message::severity_level::error, "XRT", msg);
 }
 
+static void
+configure_bd(xrtDeviceHandle dhdl, int tileType, uint8_t column, uint8_t row, uint8_t bdId,
```

Comment:
WOW .. 26 parameters.  Having 8 is pushing it but 26.

This is not a good practise at all.